src/ServerHarness/App.config

```diff
@@ -43,6 +43,9 @@
          and accept the risks of having additional attack surface. As long as FtpPort remains blank, the optional FTP
          system will not open any extra ports. -->
     <add key="FtpPort" value="" />
+
+	<!-- Controls whether the RavenDB web interface loads automatically or not. -->
```

Comment:
nit: We ended up with mixed tabs/spaces with this.

Nice.  I do like this defaulting true for now, indeed seems it could really help with ramp-up discovery.